torch/nn/parallel/distributed.py

```diff
@@ -1166,9 +1166,26 @@ def __setstate__(self, state):
             self.logger._set_static_graph()
 
     def _build_params_for_reducer(self):
+        from torch.distributed._tensor import DTensor
+        import functools
+        def _dtensor_ext(param_name, param):
+            def tensor_hook(param, grad):
```

Comment:
what's the ordering of these hooks with respect to the Reducer installed hooks? How do we guarantee this ordering?

Reply:
This is not needed, actually. I haven't cleaned it up. I need to polish this PR a little bit more.